### PR TITLE
roadmap (fixes #299)

### LINKED ISSUE #299: Publish a roadmap (and reference in README)

### PR DESCRIPTION
**Pull request by ngokevin**
Changes proposed:
- Rough preliminary roadmap

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+- Standard physics, template, text components
+- Custom geometries
+
+## Long-term
+
+- Scene editor and inspector
+- Asset discovery service
+- Multiplayer
+
+- [extensibleweb]: https://www.w3.org/community/nextweb/2013/06/11/the-extensible-web-manifesto/
```
Comment (dmarcos) [inline ROADMAP.md]: This looks out of place. Is it part of the long term, roadmap or general philosophy of the project?

Comment (ngokevin) [inline ROADMAP.md]: Fixed the link. This line isn't supposed to render.

Comment (ngokevin) [inline ROADMAP.md]: Nevermind. Just removed the link. Forgot this is a README file not a docs file

File: ROADMAP.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Roadmap
+
+## Top-Level Goals
```
Comment (cvan) [inline ROADMAP.md]: lowercase to match casing of headings below

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- Better mobile video support
+- Performance (geometry merging, asset management)
+- Standard physics, template, text components
+- Custom geometries
+
+## Long-term
+
+- Scene editor and inspector
+- Asset discovery service
+- Multiplayer
```
Comment (cvan) [inline ROADMAP.md]: `Multi-user` perhaps

**Comment by cvan** [comment]
lgtm

**Pull request merged**